UnitySDK/Assets/ML-Agents/Scripts/Startup.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace MLAgents
+{
+    public class Startup: MonoBehaviour
+    {
+        void Awake()
+        {   
+            string sceneName = Environment.GetEnvironmentVariable("SCENE_NAME");
+            SwitchScene(sceneName);
+        }
+        
+        private static void SwitchScene(string sceneName)
+        {
+            if (sceneName == null)
+            {
+                throw new ArgumentException("You didn't specified the SCENE_NAME environment variable");
+            }
+            else
```

Comment:
This else is unnecessary since the method will return when the exception in thrown. I would remove nesting as much as possible to improve readability

Reply:
Changed.